src/commands/frameworks-stacks-create.ts

```diff
@@ -4,8 +4,8 @@ import { needProjectId } from "../projectUtils";
 import requireInteractive from "../requireInteractive";
 import { doSetup } from "../init/features/frameworks";
 
-export const command = new Command("stacks:create")
-  .description("Create a stack in a Firebase project")
+export const command = new Command("backends:create")
```

Comment:
Small semantic nit, but should this be `backend:create` as opposed to `backends:create`? If we're creating an instance of a backend it seems to me the more logical choice will be the singular version of the noun as opposed to the plural form here. (same comment throughout this PR)